### ISSUE TITLE
Allow ranked maps to be unranked via beatmapsets admin panel

### ISSUE THREAD

**Issue by Ephemeralis** [ppy/osu-web#9075]
Unranks happen relatively infrequently (1-12~ times a year), but presently require someone with database access to manually set ``approved=3`` and then hit the disqualify button via beatmapset discussions in order to ensure everything ends up like it should (scores, etc). It's all kind of janky.

It would be preferable if an equivalent was available as an admin-only function in the beatmapsets admin panel, especially as further development may further complicate the manual steps required to ensure things don't break.

**Comment by peppy** [comment]
so ASS right? i don’t see any reason to put such controls on the site…

**Comment by Ephemeralis** [comment]
ASS works, yeah.

**Comment by peppy** [comment]
@Ephemeralis I guess this is still relevant? Might as well knock this command out.

**Comment by Ephemeralis** [comment]
Yep, just lock it to admin roles on Discord and we should be good.
